### ISSUE TITLE
[RFC] Image Slider/Carousel

### ISSUE THREAD

**Issue by brianespinosa** [microsoft/fluent-ui-react#405]
# Feature Request
Placeholder for discussion on a slider/carousel component

# Problem Description
Stardust does not yet have a carousel or slideshow type component. We should probably provide something with basic functionality for not only images, but any content.

# What To Include
This is an initial list of both UI and functional features we should consider

#### UI
- Left/Right navigation
- Dot strip navigation
- Film strip navigation
- Positioning for any navigation components (top/bottom)

#### Functionality
- OOTB transition options
- Continuous sliding option (next @ end shows the first item)
- Automatic building of navigation based on content added to the carousel

------
If there is interest, I actually have a very simple slider that does some of this that I could probably take a shot at implementing in the Stardust environment once there are some more solid pattern foundations here.

**Comment by levithomason** [comment]
Agreed.  I have a vanilla JS state manager prototype of this already that I'd like to port as well.  See https://codesandbox.io/s/jzjpxwjp4y for early prototypes of how we'd like to manage state in a vanilla JS way.

**Comment by robin-liao** [comment]
Any follow up on this? It's been a while. Please let us know if it's still in the plan. Thanks! @levithomason

**Comment by levithomason** [comment]
There is now a W3 guide to carousels.  We should start with a base implementation of this:

https://www.w3.org/WAI/tutorials/carousels/structure/

@jurokapsiar can your prioritize this one, please?
